blocks/editable/index.js

```diff
@@ -103,6 +107,23 @@ export default class Editable extends wp.element.Component {
 		this.props.onFocus();
 	}
 
+	isActive() {
+		return document.activeElement === this.editor.getBody();
+	}
+
+	onSelectionChange() {
+		// We must check this because selectionChange is a global event.
+		if ( ! this.isActive() ) {
+			return;
+		}
+
+		const content = this.getContent();
+
+		this.setState( {
```

Comment:
>I'm not sure if this would trigger a re-render if the value is the same?

Yes it does: https://codepen.io/aduth/pen/mmXgxQ

We'd need to expose and use `React.PureComponent` for it to not: https://facebook.github.io/react/docs/react-api.html#react.purecomponent